doc/dnf5_plugins/repoclosure.8.rst

```diff
@@ -0,0 +1,78 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _repoclosure_plugin_ref-label:
+
+####################
+ Repoclosure Command
+####################
+
+Display a list of unresolved dependencies for repositories.
+
+Synopsis
+========
+
+``dnf5 repoclosure [options] [<pkg-spec>...]``
+
+
+
+The ``repoclosure`` command in ``DNF5`` reads package metadata from one or more repositories, checks all dependencies, and displays a list of packages with unresolved dependencies.
+
+
+Options
+-------
+
+``--arch <arch>``
+    Query only packages for specified architecture, can be specified multiple times (default is all
+    compatible architectures with your system).
+
+``--best``
+    Check only the newest packages per arch.
+
+``--check <repoid>``
+    Specify repo ids to check, can be specified multiple times (default is all enabled).
+
+``--newest``
+    Check only the newest packages in the repos.
+
+``--repo <repoid>``
```

Comment:
I am thinking if this should be mentioned here. 
It is a general dnf option, for example I think `--enable-repo` would also work but it is not mentioned.

Though I see how it is relevant. :thinking:

Reply:
Yes, I guess just command-specific options should be mentioned there. But it's definitely good to have the common use cases in the examples below as it is now.

Reply:
Well to be honest, I'd like to have different set of options for repoclosure, something like:
--repo <repo to check>
--lookaside <also this repo that can be used to fulfill requirements>
But this design collides with existing `--repo` option which is consistently used in all commands and defines which repositories are available to dnf... And yes, --enable-repo will also work.

So that's why I've decided to include this global option also to repoclosure documentation. But I can try to replace the option in docs by better description of the command behavior.

Reply:
I like the idea that the specific behavior of the option used for this command is described in it's man page. I consider this locality better than describing different behaviors in the general man page, but we could still mention in the general man page there is a difference for `repoclosure`, see its man page (or something like this).